cpu.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
+
+#include <stddef.h>
```

Comment:
Remove the inclusion of `stddef.h`.

Reply:
Where should I put this header ?
‘cpu.c’ ?

Reply:
Move to header `core.h` if it is required.
You should always eliminate the inclusion of existing headers as possible.